docs/trace/stratified-sampling-example/stratified-sampling-example.csproj

```diff
@@ -1,14 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
```

Comment:
why was this PropertyGroup got removed?

Reply:
My goal is that all projects under `/docs` folder should inherit the default setting, and avoid any violations (because anything inconsistent will make it harder to maintain). Specifically to this change, I think all of these are inherited from the common project/folder.